tests/dataplane-performance/results/1.0.0.md

```diff
@@ -0,0 +1,105 @@
+# Results
+
+## Test environment
+
+GKE cluster:
+
+- Node count: 3
+- Instance Type: e2-medium
+- k8s version: 1.27.4-gke.900
+- Zone: europe-west2-b
+- Total vCPUs: 6
+- Total RAM: 12GB
+- Max pods per node: 110
+
+NGF deployment:
+
+- NGF version: edge - git commit a41e9e46d72788bceea8ba44e01d4062afec75fd
+- NGINX Version: 1.25.2
+
+## Test1: Running latte path based routing
+
+Running 30s test @ http://cafe.example.com/latte
+  2 threads and 10 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency    33.17ms    3.79ms  50.28ms   69.72%
+    Req/Sec   150.93     14.21   202.00     80.27%
+  Latency Distribution
+     50%   33.96ms
+     75%   35.81ms
+     90%   37.21ms
+     99%   41.29ms
+  9041 requests in 30.07s, 3.22MB read
+Requests/sec:    300.71
+Transfer/sec:    109.54KB
+
+## Test2: Running coffee header based routing
+
+Running 30s test @ http://cafe.example.com/coffee
+  2 threads and 10 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency    32.46ms    3.87ms  59.48ms   68.58%
+    Req/Sec   154.29     15.66   202.00     75.42%
+  Latency Distribution
+     50%   33.38ms
+     75%   35.22ms
+     90%   36.65ms
+     99%   40.23ms
+  9239 requests in 30.07s, 2.71MB read
+  Non-2xx or 3xx responses: 9239
```

Comment:
```
 9239 requests in 30.07s, 2.71MB read
  Non-2xx or 3xx responses: 9239
```

You probably got 404s here from NGF, rather than responses from the backend

Reply:
I had a typo in my wrk command 🤦🏼‍♀️